mysql-schema/mysql-schema.md

```diff
@@ -117,6 +117,7 @@ Currently, the `help_topic` is NULL.
 ## System tables related to backup and restore
 
 * `tidb_pitr_id_map`: the ID mapping information for point-in-time recovery (PITR) operations
+* `tidb_restore_registry`: the registry for restore tasks with unique restore IDs assigned to prevent different restore operations from overlapping with each other
```

Comment:
```suggestion
* `tidb_restore_registry`: the registry for restore tasks with unique restore IDs assigned to prevent conflicts between different restore operations.
```